plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestParquetPageSkipping.java

```diff
@@ -88,6 +88,27 @@ public void testAndPredicates()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testRandomData()
```

Comment:
Give it a meaningful name. What makes this test worthy keeping?

Reply:
updated name and added a comment